src/ameba/inline_comments.cr

```diff
@@ -1,7 +1,7 @@
 module Ameba
   # A module that utilizes inline comments parsing and processing logic.
   module InlineComments
-    COMMENT_DIRECTIVE_REGEX = Regex.new "# ameba : (\\w+) ([\\w/, ]+)".gsub(" ", "\\s*")
+    COMMENT_DIRECTIVE_REGEX = Regex.new "# ameba : (\\w+) ([\\w/, ]*)".gsub(" ", "\\s*")
```

Comment:
I'd suggest a pattern that's a lil' bit more strict:

```suggestion
    COMMENT_DIRECTIVE_REGEX = /# ameba:(?<action>\w+) (?<rules>\w+(?:, \w+)*)/
```

Reply:
This will break the backward compatibility which I wouldn't want to do

```crystal
# ameba : disable AnyRule
```

will not disable, neither reported.

Reply:
That's true, but honestly I haven't seen this notation in a wild - and it's weird anyway, why would you write it like that? I don't think any other linters I know supports that either.

Reply:
Why does having a space in the notation look weird to you?

Reply:
It seems unnecessary, less concise and breaks up visual flow, which otherwise is, at least for me more consistent - `ameba:disable` - pragma and then whatever comes next are the rules/groups vs `ameba` -> `disable` and then rules/groups. Also, as I said before, all other linters use such notation so there's one thing less to "learn".

Reply:
Thanks. Okay, i will change it in the next PR.